runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -361,25 +361,28 @@ std::shared_ptr<exec::ExecutorMap> Compiler::compile(void)
   /***************************************************
    * Prepare compilation phase
    ***************************************************/
-  auto executors = std::make_shared<exec::ExecutorMap>();
-
   // Compilable check
   // TODO: Support hybrid execution -
   //       execution between interpreter and compiled executor (including control flow)
   if (_options.disable_compile)
   {
+    auto executors = std::make_shared<exec::ExecutorMap>();
+
     _subgraphs->iterate([&](const ir::SubgraphIndex &index, ir::Graph &subg) {
       executors->emplace(index, std::make_unique<interp::InterpExecutor>(subg));
       subg.setSubgraphs(_subgraphs);
     });
     _state = State::COMPILED;
-    return executors;
+    return std::make_shared<CompilerArtifact>(executors, nullptr);
   }
 
   // Mode check
   if (_options.he_profiling_mode)
     checkProfilerConditions();
 
+  // Tracing context
+  auto tracing_ctx = std::make_unique<util::TracingCtx>(_subgraphs.get());
```

Comment:
@hseok-oh (beyond this PR) What about creating `tracing_ctx` only tracing is enabled (i.e. `TRACEFILE_PATH` != nullptr)?

Reply:
Good. I'll update later.